torchvision/prototype/transforms/_color.py

```diff
@@ -82,6 +82,7 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return output
 
 
+# TODO: Are there tests for this class?
```

Comment:
I don't think so. We have a pretty good grip on the functional testing, but so far the transforms testing is the wild west. Seems we have forgotten this completely.